### PR TITLE
Portfolio accessibility check

### PR DESCRIPTION
**Pull request by Isaac-ej**
In this feature, we have evaluated our websites against accessibility best practices and implemented some necessary changes to ensure that our website is accessible.

All the points we have checked so far are below:

Page titles: no issues found.
 Image text alternatives: issue found and here is the link, https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL46C2-L46C2
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L59
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL50C4-L50C132
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL53C9-L53C77
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL56C3-L56C3
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL59C12-L59C103
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL68C7-L68C77
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL68C7-L68C77
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L105
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L140
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L254
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L256
https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L259
Text Heading: No issue found.
Color Contrast: issue found. 
- https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L281
- https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL282C6-L282C131
- https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L283
- https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#L41
- https://github.com/Isaac-ej/Portfolio_Website/blob/website_accessibility/index.html#LL92C1-L97C2
Resize: No issue found.
Interaction: No issue found.
Moving content: No issue found.
Multimedia: No issue found.
The basic structure of the page: No issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,22 +41,23 @@ <h1>I'm Isaac. <span class="span">Glad To see you!</span> </h1>
         <h2>LET'S CONNECT</h2>
```
Comment (skyv26) [inline index.html:41]: - [ ] Kindly consider addressing the text colour contrast issue on the portfolio. It affects readability for users, particularly those with visual impairments. Enhancing contrast ratios, seeking diverse user feedback, and providing customizable options can improve inclusivity and user satisfaction. Please use [this](https://coolors.co/contrast-checker) website to match the text color contrast with the background.

File: index.html
Comment (skyv26) [inline index.html:1]: - [ ] Kindly consider addressing the text colour contrast issue on the portfolio. It affects readability for users, particularly those with visual impairments. Enhancing contrast ratios, seeking diverse user feedback, and providing customizable options can improve inclusivity and user satisfaction. Please use [this](https://coolors.co/contrast-checker) website to match the text color contrast with the background. 

```
 <li><span>HTML</span> </li>
<li><span>CSS</span> </li>
<li><span>Javascript</span> </li>
```

Please do make changes by adjusting the color for your project cards skills.

Comment (skyv26) [inline index.html:1]: - [ ] Kindly 🙏🏻  consider implementing the use of `aria-label` in input tags on the platform. This attribute provides a text description directly within the input tag, improving accessibility for users relying on assistive technologies. For instance, <input type="text" aria-label="Enter your name" placeholder="Name"> clearly conveys the purpose of the input field without the need for a separate label. Please prioritize this enhancement for a more inclusive user experience.

```sh
 <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
  <input type="email" name="email" id="email" placeholder="Alexandra@gmail.com" required>
  <textarea name="msg" id="msg" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>
```

Please add the **aria-label** in above code.

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @Isaac-ej and @Cheptoo21, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @Isaac-ej  and @Cheptoo21 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ contrasts are corrected
✔️ form input fields are corrected
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**